pakyow-presenter/lib/presenter/binder.rb

```diff
@@ -59,16 +59,29 @@ def value_for_scoped_prop(scope, prop, bindable, bindings, context)
 
         if binding_fn
           binding_eval = BindingEval.new(prop, bindable, context)
-          binding_eval.instance_exec(binding_eval.value, bindable, context, &binding_fn)
-        else # default value
-          if bindable.is_a?(Hash)
-            bindable.fetch(prop) { bindable[prop.to_s] }
-          elsif bindable.class.method_defined?(prop)
-            bindable.send(prop)
+          result = binding_eval.eval(&binding_fn)
+
+          # TODO Decide if bindings given as arguments should also only use
```

Comment:
I think we can do it gradually. I will add some code to allow bindings to also use a hash. Something to decide now is what the migration path would look like and whether warnings should be added or just some notes in the docs.